libc/test/src/math/performance_testing/BinaryOpSingleOutputPerf.h

```diff
@@ -25,21 +26,28 @@ template <typename T> class BinaryOpSingleOutputPerf {
 
   static void run_perf_in_range(Func myFunc, Func otherFunc,
                                 StorageType startingBit, StorageType endingBit,
-                                StorageType N, std::ofstream &log) {
+                                size_t N, size_t rounds, std::ofstream &log) {
+    if (endingBit - startingBit < N)
+      N = endingBit - startingBit;
+
+    size_t total_ops = N * rounds;
```

Comment:
Hopes that it will never overflow.

Reply:
or you can do `/ N / rounds` and don't have to hope.